### DISCUSSION TITLE
Course 1 Lesson 2 - Conflict, Political Science & Information Flows - Missing A Clip

### DISCUSSION THREAD

**Discussion by nahuakang** [OpenMined/courses#80] [Course: Our Privacy Opportunity] [answered]
[This video](https://youtu.be/xIn54pBRHyA) from 11:53 to 11:58 misses a clip of a real-world example of a 3rd party facilitating the mediation to avoid nuclear actions from India and Pakistan.

**Comment by nahuakang** [comment]
@mcleonard

**Reply by yemikifouly** [reply]
Great! You tagged the right person too. I might add @iamtrask and @em-blue.

**Comment by iamtrask** [comment, marked as answer]
On it!
